ipn/ipnlocal/serve.go

```diff
@@ -254,10 +268,10 @@ func (b *LocalBackend) setServeConfigLocked(config *ipn.ServeConfig) error {
 
 // ServeConfig provides a view of the current serve mappings.
 // If serving is not configured, the returned view is not Valid.
-func (b *LocalBackend) ServeConfig() ipn.ServeConfigView {
+func (b *LocalBackend) ServeConfig() (ipn.ServeConfigView, string) {
```

Comment:
undocumented string result that's not obvious from its type

Reply:
Removing per comment below